extensions/amp-live-list/amp-live-list.md

```diff
@@ -24,6 +24,8 @@ limitations under the License.
 
 # amp-live-list
 
+Warning: Currently, content served via an AMP cache will be slow to receive live updates. Read more [here](https://github.com/ampproject/amphtml/issues/13659)
```

Comment:
@twifkak – Good to know so we can replace this when the fix comes out.

Reply:
Thanks for the heads up. I would s/will/may/ since the update latency depends on a bunch of factors including the AMP cache, the URL, and the response headers. But otherwise LGTM.

Reply:
fixed in 8769754